zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -587,6 +587,29 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /decision-requirements/search:
+    post:
+      tags:
+        - Decision Requirements
+      summary: Decision Requirements Search Query
```

Comment:
@tmetzke reworked the endpoints summary here https://github.com/camunda/camunda/pull/20807/files

This should be changed to `Query decision requirements (experimental)`